ide/api.java.classpath/src/org/netbeans/api/java/classpath/package.html

```diff
@@ -192,7 +192,7 @@ <h2><a name="change-listen">Listening to changes in ClassPath</a></h2>
 <a href="ClassPath.html"><font class="type">ClassPath</font></a> <font class="variable-name">cp</font> = ClassPath.<a href="ClassPath.html#getClassPath-org.openide.filesystems.FileObject-java.lang.String-"><font class="function-name">getClassPath</font></a>(f, ClassPath.<a href="ClassPath.html#COMPILE"><font class="constant">COMPILE</font></a>);
 cp.<a href="ClassPath.html#addPropertyChangeListener-java.beans.PropertyChangeListener-"><font class="function-name">addPropertyChangeListener</font></a>(new <a href="@JDK@/java/beans/PropertyChangeListener.html"><font class="type">PropertyChangeListener</font></a>() {
     <font class="keyword">public void</font> <font class="function-name">propertyChange</font>(<a href="@JDK@/java/beans/PropertyChangeEvent.html"><font class="type">PropertyChangeEvent</font></a> <font class="variable-name">evt</font>) {
-        <font class="keyword">if</font> (ClassPath.<a href="ClassPath.html#PROP_ROOTS"><font class="constant">PROP_ROOTS</font></a>.equals(evt.<a href="@JDK@/java/beans/PropertyChangeEvent.html#getPropertyName()"><font class="function-name">getPropertyName()</font></a>)) {
+        <font class="keyword">if</font> (ClassPath.<a href="ClassPath.html#PROP_ROOTS"><font class="constant">PROP_ROOTS</font></a>.equals(evt.<a href="@JDK@/java/beans/PropertyChangeEvent.html#getPropertyName--"><font class="function-name">getPropertyName()</font></a>)) {
```

Comment:
just a question: what does the `--` do? Couldn't find anything about it.

Reply:
it's a bit of anticipation for this one but the anchor in javadoc is changing parenthesis to -.

Reply:
ah. so its like a TODO for you? :)

Reply:
it's part of the phase 2.
Phase 1 is the clean up of javadoc itself to have no error and get all javadoc link ok (otherwise they need to move to our own management of modules/jdk links with replace.
Then in phase 2 the resulting checkstyle of apidoc need to be resolved to be sure to link to allowed url. each module have this check but in fact there is forward mention the only full apidoc check is important.

Reply:
like the marvel universe just with javadoc basically

Reply:
https://ci-builds.apache.org/job/Netbeans/job/netbeans-TLP/job/netbeans/job/master/lastCompletedBuild/testReport/org.netbeans.nbbuild/CheckLinks/testBrokenLinks/